data/mluhalle/oa_publicationen_mlu_halle_2.csv

```diff
@@ -0,0 +1,19 @@
+id,institution,period,euro,doi,is_hybrid,publisher,journal_full_title;;;;;;;;;;;;;
+1,Martin Luther Universit�t Halle-Wittenberg,2017,2142,10.1186/s12913-017-2456-0,FALSE,BioMed Central,BMC Health Services Research;;;;;;;;;;;;;
+2,Martin Luther Universit�t Halle-Wittenberg,2017,2148,10.3389/fpls.2017.01135,FALSE,Frontiers Media S.A.,Frontiers In Plant Science;;;;;;;;;;;;;
+3,Martin Luther Universit�t Halle-Wittenberg,2017,1874,10.3390/ijms18091844,FALSE,MDPI AG,International Journal of Molecular Sciences;;;;;;;;;;;;;
+4,Martin Luther Universit�t Halle-Wittenberg,2017,1378,10.3390/polym9110573,FALSE,MDPI AG,Polymers;;;;;;;;;;;;;
+5,Martin Luther Universit�t Halle-Wittenberg,2017,1453,10.3390/su9081486,false,MDPI AG,Sustainability;;;;;;;;;;;;;
+6,Martin Luther Universit�t Halle-Wittenberg,2017,3427,10.1093/nar/gkx931,false,Oxford University Press (OUP),Nucleic Acids Research;;;;;;;;;;;;;
+7,Martin Luther Universit�t Halle-Wittenberg,2017,2332,10.1159/000481732,false,S. Karger AG,Obesity Facts;;;;;;;;;;;;;
+8,Martin Luther Universit�t Halle-Wittenberg,2017,2148,10.3389/micb.2017.02180,false,Frontiers Media S.A.,Frontiers in Microbiology;;;;;;;;;;;;;

```

Comment:
Many thanks for contribution! There are two issues, however:
- This DOI does [not resolve](https://doi.org/10.3389/micb.2017.02180)
- We need a contact person and a mail address for the MLU. 

You might want to get into contact with us at openapc at uni-bielefeld de